packages/link/src/Link.ts

```diff
@@ -347,6 +304,21 @@ async function eventsListener(
   }
 }
 
+function handleErrorAndSendMessage(error: unknown, messageType: string) {
+  let errorMessage = 'An unexpected error occurred'
+  if (error instanceof Error) {
+    errorMessage = error.message
+  } else {
+    console.error('An unexpected error occurred', error)
```

Comment:
Please don't print anything to the console. If you have this at other places, pls remove that